### PR TITLE
feat(account): implement account create sub command

### PR DESCRIPTION
**Pull request by realdreamer**
https://app.asana.com/0/1205969628270714/1206644161687868/f

account creation on only `mainnet`

`kadena account create`

### CODE REVIEW

File: packages/tools/kadena-cli/src/account/utils/createAccountOnMainnet.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    const networkConfig = networkDefaults.mainnet;
+    const account = isNotEmptyString(accountName)
+      ? accountName
+      : await createAccountName({
+          publicKeys,
+          chainId,
+          predicate,
+          networkConfig,
+        });
+    const keyPair = genKeyPair();
```
Comment (nillo) [inline packages/tools/kadena-cli/src/account/utils/createAccountOnMainnet.ts:36]: what does this do in this context

File: packages/tools/kadena-cli/src/account/utils/createAccountOnMainnet.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          networkId: networkConfig.networkId,
+          meta: {
+            chainId: chainId,
+            gasLimit: 840,
+          },
+        },
+        sign: createSignWithKeypair([keyPair]),
+      },
+    )
+      .on('submit', (data) => data)
```
Comment (nillo) [inline packages/tools/kadena-cli/src/account/utils/createAccountOnMainnet.ts:64]: ?

File: packages/tools/kadena-cli/src/account/utils/createAccountOnMainnet.ts (hunk 1 above)
Comment (realdreamer) [reply packages/tools/kadena-cli/src/account/utils/createAccountOnMainnet.ts:36]: this keypair only does signing

**Review by barthuijgen — APPROVED** [review]

**Review by nillo — APPROVED** [review]

File: packages/tools/kadena-cli/src/account/utils/createAccountOnMainnet.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    const networkConfig = networkDefaults.mainnet;
+    const account = isNotEmptyString(accountName)
+      ? accountName
+      : await createAccountName({
+          publicKeys,
+          chainId,
+          predicate,
+          networkConfig,
+        });
+    const keyPair = genKeyPair();
```
Comment (nillo) [inline packages/tools/kadena-cli/src/account/utils/createAccountOnMainnet.ts:36]: @jermaine150

**Pull request merged**